Readme.md

```diff
@@ -1,37 +1,40 @@
 # ddprof
 
-The Datadog Native Profiler for Linux
+The Datadog Native Profiler for Linux.
 
 ## Overview
 
-`ddprof` is a commandline utility for engaging kernel-mediated telemetry of an application and forwarding the resulting information to the Datadog backend.  In several ways, it's similar to the `perf record` tool.
+`ddprof` is a command-line utility to gather profiling data. After install you will continuously see where your application is spending CPU and memory allocations.
+The data will be available in the `/profiling` section of the Datadog UI.
 
 ## Quick Start
 
 Our official documentation is available [here](https://docs.datadoghq.com/profiler/enabling/ddprof/?tab=environmentvariables).
+Our pre-built binaries are compatible with both musl and glibc. You should not need to recompile `ddprof` from source. 
 
-### From binary
+### From binary [Recommended]
 
-Check out our Release page for prebuilt binaries. Download the desired binary, making sure to mark it executable `chmod +x ./ddprof`.
-Refer to [commands](docs/Commands.md) for the commands supported by `ddprof`. Example :
+A full installation guide is available [here](https://docs.datadoghq.com/profiler/enabling/ddprof/?tab=environmentvariables).
+Check out our Release page for our latest release. Download the release and extract `ddprof`.
+Instrumenting your application should be as simple as adding `./ddprof` in front of your usual command line.
 
 ```bash
-./ddprof -S my_native_service ./run.sh
+./ddprof -S service_name_for_my_program ./my_program arg1 arg2
 ```
 
+Profiling data shows up in the `/profiling` section of your Datadog UI. Specifying a service name will help you select your profiling data. 
+Refer to [commands](docs/Commands.md) for a more advanced usage of `ddprof`.
+
 ### From source
 
 Checkout our build section [here](./docs/Build.md).
 
 ### Prerequisites
 
-The Datadog Native Profiler for Linux has only been tested on kernel 4.15 above.  It may be supported by older kernels, but your mileage may vary.  One can verify the kernel version by running `uname`:
-
-```bash
-uname -r
-```
+#### Perf event paranoid
 
-In addition, the target machine must have `perf_event_paranoid` set to 2 or lower OR `CAP_SYS_ADMIN` enabled.
+The target machine must have `perf_event_paranoid` set to 2 or lower OR `CAP_SYS_ADMIN` enabled.
+Some distributions have set the default to higher values (up to 4), which will prevent `ddprof` from instrumenting CPU activity through the perf_event_open API.
```

Comment:
Maybe add a bash snippet to change the value `perf_event_paranoid` ?

Reply:
I don't know if we need to specify that some distros have higher values.  I feel that `perf_event_paranoid <= 2` covers that and wonder whether overstating it might be confusing for some people.

We should probably mention `CAP_PERFMON` being introduced in Linux 5.8.

Reply:
I thought of this, though it would not cover all cases. I guess the most common way is to have a boot time setting ?

`sudo sh -c 'echo kernel.perf_event_paranoid=2 > /etc/sysctl.d/perf_event_paranoid_2.conf'`

Reply:
I was more thinking about changing `perf_event_paranoid` temporarily for testing with:
```bash
echo 2 | sudo tee /proc/sys/kernel/perf_event_paranoid
```
But both would be nice